graphistry/pygraphistry.py

```diff
@@ -270,8 +271,10 @@ def protocol(value=None):
     def api_version(value=None):
         """Set or get the API version: 1 or 2 for 1.0 (deprecated), 3 for 2.0
         Also set via environment variable GRAPHISTRY_API_VERSION."""
-        if value is None:
```

Comment:
I think we still need `:273` for the getter mode...

Reply:
Currently when the value is `None`, that is, not set by the user, `api` is set to `1` (default config). Do you expect that we keep this behaviour?

Reply:
Yes we should preserve -- that will work in some environments that set params via env vars, so rejecting would be a breaking change for them . When we start defaulting to api=3, that'd be a friendlier time to reject.

The current need is just to catch people accidentally passing in say `4` or `"3"`